### PR TITLE
Write an advanced usage section

### PR DESCRIPTION
**Pull request by Adamantios**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,3 +144,45 @@ Execute the following steps in a PowerShell terminal:
...
+[here](https://github.com/valory-xyz/trader-quickstart/blob/0f093ebbf0857b8484a017912c3992f00fbe1a29/run_service.sh#L698), 
+in order to set your own custom warm start. 
+Setting your own custom weights can be done by editing the corresponding files in `.trader_runner`.
+Moreover, you may store your current policy as a backup before editing those files, using the following command:
+
+```shell
+cp ".trader_runner/available_tools_store.json" ".trader_runner/available_tools_store_$(date +"%d-%m-%Y")".json
+```
+
+##### Tool selection
```
Comment (DavidMinarsch) [inline README.md:174]: Step 1 should be to tell the user about the env variable that adjusts the policy punishment

File: README.md [hunk 2]
```diff
@@ -144,3 +144,45 @@ Execute the following steps in a PowerShell terminal:
...
+These data were obtained after many days of running the service and are set 
+[here](https://github.com/valory-xyz/trader-quickstart/blob/0f093ebbf0857b8484a017912c3992f00fbe1a29/run_service.sh#L133-L137).
+As a result, the current weights are always deleted and replaced by this strategy 
+which is considered to boost the initial performance of the service.
+
+However, you may have found better performing policy weights and would like to remove this logic. 
+It can easily be done, by removing this method call, 
+[here](https://github.com/valory-xyz/trader-quickstart/blob/0f093ebbf0857b8484a017912c3992f00fbe1a29/run_service.sh#L698), 
+in order to set your own custom warm start. 
+Setting your own custom weights can be done by editing the corresponding files in `.trader_runner`.
```
Comment (DavidMinarsch) [inline README.md:167]: Why can we not simply say: if you want a different warm start adjust such and such file?

File: README.md (hunk 1 above)
Comment (Adamantios) [reply README.md:174]: We do not have an environment variable for the punishment, this is fixed to the mech's price.

File: README.md (hunk 2 above)
Comment (Adamantios) [reply README.md:167]: Because the method call needs to be removed as well, to avoid overwritting the files' contents.

File: README.md (hunk 1 above)
Comment (dagacha) [reply README.md:172]: Open an issue https://github.com/valory-xyz/trader/issues/120

I suggest we close this comment.

**Pull request merged**